Input/InputSpotlightDecorator.js

```diff
@@ -27,6 +27,23 @@ const handleKeyDown = handle(
 	call('onKeyDown')
 );
 
+/**
+ * Default config for [InputSpotlightDecorator]{@link agate/Input.InputSpotlightDecorator}
```

Comment:
We might need the `.` at the end of the line.

Reply:
Sure!